consensus/ConsensusFileGenerator.py

```diff
@@ -108,42 +108,42 @@ def _add_history_of_variant(id_to_match, export, variant_history):
     def _get_history_ids_for_variant(variant_id, chromosome, position, ref, alt, gene, variant_type):
```

Comment:
Why not make "_get_history_ids_for_variant()" non-static so you can use self instead of ConsensusFileGenerator._function_